### DISCUSSION TITLE
Usage on Pi Zero - how to optimize aircast?

### DISCUSSION THREAD

**Discussion by unlikeother** [philippe44/AirConnect#373] [Q&A]
Hello,

I´m using your great app on a Pi Zero, so that I can stream Apple Music to a pair of Google Home Minis (v1). I got it working, but audio is not stable - is there any config I can change so that I can optimize this?

**Comment by pwt** [comment]
What are you using for the latency/buffering (`-l`) parameter?

**Comment by unlikeother** [comment]
I didn´t change anything...so nothing, what do you recommend??

pwt ***@***.***> escreveu no dia quarta, 6/10/2021 à(s) 14:18:

**Reply by pwt** [reply]
> I didn´t change anything...so nothing, what do you recommend?

Try `-l 1000:1000:f` to start with. That's what my container version uses.

**Comment by unlikeother** [comment]
Thank you, will try it when i get home!

pwt ***@***.***> escreveu no dia quarta, 6/10/2021 à(s) 14:28:

**Comment by unlikeother** [comment]
It worked! Thanks

antonioj ***@***.***> escreveu em qua., 6/10/2021 às 14:28 :
